### PR TITLE
Increment MIT Licence copyright header year on files modified in 2017

### PR DESCRIPTION
**Pull request by ghost**
Edited via:

$ contrib/devtools/copyright_header.py update .

ps) It is the same commit as #9450

### CODE REVIEW

File: src/univalue/test/object.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2014 BitPay Inc.
-// Copyright (c) 2014-2016 The Bitcoin Core developers
+// Copyright (c) 2014-2017 The Bitcoin Core developers
```
Comment (fanquake) [inline src/univalue/test/object.cpp]: This file shouldn't be modified, as it's part of the univalue subtree. I've updated our copyright script in #12075.

Comment (akx20000a) [reply src/univalue/test/object.cpp]: Edited, thanks!

Comment (herojab) [reply src/univalue/test/object.cpp]: #12075

**Comment by zcheng7905** [comment]
hi,
   Do you wanna crack bitcoin? the gating is ECC（Elliptic Curves Cryptography)and I have a nice way to crack ECC,Are you interested?



zcheng_7905@163.com
 
From: Akira Takizawa
Date: 2018-01-02 11:05
To: bitcoin/bitcoin
CC: Subscribed
Subject: Re: [bitcoin/bitcoin] Increment MIT Licence copyright header year on files modified in 2017 (#12062)
@akx20000a commented on this pull request.


In src/univalue/test/object.cpp:
> @@ -1,5 +1,5 @@
 // Copyright (c) 2014 BitPay Inc.
-// Copyright (c) 2014-2016 The Bitcoin Core developers
+// Copyright (c) 2014-2017 The Bitcoin Core developers

Edited, thanks!

**Comment by MarcoFalke** [comment]
Please squash your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by akx20000a** [comment]
@MarcoFalke  Done

**Comment by laanwj** [comment]
Verified that the changes in this PR match the output of that command.
utACK 595a7bab23bc21049526229054ea1fff1a29c0bf

**Pull request merged**